src/media/css/elements--categories.styl

```diff
@@ -49,6 +49,9 @@ mkt-category-item {
         display: inline-block;
         margin: 0 10px 0 0;
         vertical-align: middle;
```

Comment:
I'm a bit torn on this. The styles are for this element so I think it makes sense to include them right after, but I think we'd leave a blank line for something like `&.is-active` so I should probably move it down.